app/sections/image-with-text.tsx

```diff
@@ -0,0 +1,191 @@
+import type {
+  HydrogenComponentProps,
+  HydrogenComponentSchema,
+} from '@weaverse/hydrogen';
+import { forwardRef } from 'react';
+import { CSSProperties } from 'react';
+import { Image } from '@shopify/hydrogen';
+
+
+interface ImageWithTextProps extends HydrogenComponentProps {
+  imageFisrt: {
+    url: string;
+    altText: string;
+    width?: number;
+    height?: number;
+  }
+  imageSecond: {
+    url: string;
+    altText: string;
+    width?: number;
+    height?: number;
+  }
+  heading: string;
```

Comment:
Create child components for Heading, Description, Actions (with Button as a child)